README.md

```diff
@@ -411,6 +439,54 @@ When applying the constraint using `kubectl apply -f constraint.yaml` with a Con
 
 To find the error, run `kubectl get -f [CONSTRAINT_FILENAME].yaml -oyaml`. Build errors are shown in the `status` field.
 
+### Customizing Admission Behavior
+
+Gatekeeper is a [Kubernetes admission webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#webhook-configuration)
+whose default configuration can be found in the `gatekeeper.yaml` manifest file. By default, it is
+a `ValidatingWebhookConfiguration` resource named `gatekeeper-validating-webhook-configuration`.
+
+Currently the configuration specifies two webhooks: one for checking a request against
+the installed constraints and a second webhook for checking the labels on namespace requests.
```

Comment:
`a second webhook for validating labels on namespace requests for the purpose of bypassing constraints for that namespace.`

Reply:
added: "... that would result in bypassing constraints for the namespace."